src/frontend/Sider/Behandling/Venstremeny/Dokumentoversikt/Dokumentoversikt.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { useEffect, useState } from 'react';
+
+import Dokumentliste from './Dokumentliste';
+import { useApp } from '../../../../context/AppContext';
+import { useBehandling } from '../../../../context/BehandlingContext';
+import DataViewer from '../../../../komponenter/DataViewer';
+import { relevanteArkivtemaer } from '../../../../typer/arkivtema';
+import { DokumentInfo, VedleggRequest } from '../../../../typer/dokument';
+import { Ressurs, byggTomRessurs } from '../../../../typer/ressurs';
+
+const Dokumentoversikt: React.FC = () => {
+    const { request } = useApp();
+    const { behandling } = useBehandling();
+
+    const [dokumenter, settDokumenter] = useState<Ressurs<DokumentInfo[]>>(byggTomRessurs());
+
+    useEffect(() => {
+        request<DokumentInfo[], VedleggRequest>(
+            `/api/sak/vedlegg/fagsak-person/${behandling.fagsakPersonId}`,
+            'POST',
+            {
+                tema: relevanteArkivtemaer,
```

Comment:
Har egt ikke spurt noen om hvilke temaer som bør hentes her 🤓

Reply:
Spurte nå, man ville ha de + kontantstøtte

Reply:
Risikerer vi ikke å hente veldig mange dokumenter her då?
I EF viser man kun dokumentet til ENF inne på behandlingen. 

Når jeg tidligere satt med Tone så hadde de personer vi såg på flere sider med dokumenter koblet til andre saker, som vanligtvis ikke er interessant for vår behandling. 

Hva med å kun vise våre dokumenter, og en lenke til dokumentoversikten for å vise alle dokumenter på personen?